docker/jukebox.Dockerfile

```diff
@@ -49,8 +49,7 @@ RUN [ "$(uname -m)" = "aarch64" ] && ARCH="arm64" || ARCH="$(uname -m)"; \
     tar -xzf libzmq.tar.gz -C ${ZMQ_PREFIX}; \
     rm -f libzmq.tar.gz;
 
-RUN export ZMQ_PREFIX=${PREFIX} && export ZMQ_DRAFT_API=1
-RUN pip install -v --no-binary pyzmq --pre pyzmq
+RUN export ZMQ_PREFIX=${ZMQ_PREFIX} && export ZMQ_DRAFT_API=1 && pip install pyzmq --pre pyzmq
```

Comment:
This code shouldn't differ on any host env. Just tested this on my Mac, and it does not work. It installs 4.3.4

```
jukebox  | 03.01.2024 22:09:06 -   67:server.py          - jb.rpc.server        - MainThread      - INFO     - Init RPC Server (Pyzmq version: 25.1.2; ZMQ version: 4.3.4; has draft API: False)
```

The current version of the code installs the correct version

https://github.com/MiczFlor/RPi-Jukebox-RFID/blob/8ea49cf49508f389d6ba83d50787539950532e79/docker/jukebox.Dockerfile#L47-L48

```
jukebox  | 03.01.2024 22:11:40 -   67:server.py          - jb.rpc.server        - MainThread      - INFO     - Init RPC Server (Pyzmq version: 25.1.2; ZMQ version: 4.3.5; has draft API: True)
```

Will try on a Windows machine, and WSL too.

Reply:
Btw. it's the same code we run during the installation itself. 

https://github.com/MiczFlor/RPi-Jukebox-RFID/blob/935a6d471e7638d18fe90358d30aba2c4cb656ac/installation/routines/setup_jukebox_core.sh#L111-L112

These commands should not differ

Reply:
Btw. Combining both `RUN` commands into a single line did not fail while building them, but running them did not work.

Same error as above

```
jukebox  | ImportError: /root/RPi-Jukebox-RFID/.venv/lib/python3.9/site-packages/zmq/backend/cython/socket.cpython-39-aarch64-linux-gnu.so: undefined symbol: zmq_join
```

Reply:
really interested, what you get on Windows. 2 RUN commands does not work for me, when running

Reply:
I have the same, even for `wsl docker-compose` ... 🤔

Reply:
Commands in one line work now.